mne/viz/ica.py

```diff
@@ -1128,15 +1123,23 @@ def plot_ica_overlay(
     return fig
 
 
-def _plot_ica_overlay_raw(data, data_cln, times, title, ch_types_used, show):
+def _plot_ica_overlay_raw(*, raw, raw_cln, picks, start, stop, title, show):
     """Plot evoked after and before ICA cleaning.
 
     Parameters
     ----------
-    ica : instance of mne.preprocessing.ICA
-        The ICA object.
-    epochs : instance of mne.Epochs
-        The Epochs to be regarded.
+    raw : Raw
+        Raw data before exclusion of ICs.
+    raw_cln : Raw
+        Data after exclusion of ICs.
+    picks : array of shape (n_channels_selected,)
+        Array of selected channels indices.
+    start : float
+        Start time to plot. Default to 0 seconds.
+    stop : float
+        Stop time to plot. Default to 3 seconds.
```

Comment:
```suggestion
    start : float
        Start time to plot.
    stop : float
        Stop time to plot.
```

there is no default, the "default" value was set in the calling function.